src/client-library/deriv-api-client-legacy.ts

```diff
@@ -7,16 +7,48 @@ type WebsocketMap = Map<string, typeof DerivAPI>;
 export class DerivAPILegacy {
     activeSocket: string;
     websocketMap: WebsocketMap = new Map();
+    keepAliveIntervalId: NodeJS.Timeout | null = null;
```

Comment:
Nit @thisyahlen-deriv, do we need this here if we're explicitly setting it to null? 
```suggestion
    keepAliveIntervalId: NodeJS.Timeout = null;
```

Reply:
Yes @jim-deriv. I believe is not just setting keepAliveIntervalId to null, it's also declaring its type

Reply:
Yeah, I just checked it, it's not inferred.